products/BellHybrid/services/time/include/time/AlarmOperations.hpp

```diff
@@ -65,14 +65,15 @@ namespace alarms
         explicit PreWakeUp(std::unique_ptr<PreWakeUpSettingsProvider> &&settingsProvider);
         auto decide(TimePoint now, const SingleEventRecord &event) -> Decision;
         auto isActive() const -> bool;
+        auto active(bool state) -> void;
```

Comment:
I would rename this function to setActive() or something similar to be more descriptive of what it does.
Alternatively, you could introduce two functions: activate() and deactivate()

Reply:
Thanks, it's a good idea. Done